src/libraries/System.Private.CoreLib/src/System/ObjectDisposedException.cs

```diff
@@ -44,6 +45,14 @@ protected ObjectDisposedException(SerializationInfo info, StreamingContext conte
             _objectName = info.GetString("ObjectName");
         }
 
+        [DoesNotReturn]
```

Comment:
Should they also have `[StackTraceHidden]`? Otherwise, the first `Throw` method should not call the second, for a cleaner stacktrace.